app/models/game.rb

```diff
@@ -31,64 +31,60 @@ def populate_black_side
 
   def populate_white_pawns
     8.times do |column|
-      # self.pieces.create(type: 'Pawn', x_position: column+1, y_position: 2)
-      # Pawn.create(game_id: self.id, x_position: column+1, y_position: 2)
-      self.pawns.create(x_position: column+1, y_position: 2)
+      self.pawns.create(x_position: column+1, y_position: 2, player_id: white_player_id)
     end
   end
 
   def populate_black_pawns
     8.times do |column|
-      # self.pieces.create(type: 'Pawn', x_position: column+1, y_position: 2)
-      # Pawn.create(game_id: self.id, x_position: column+1, y_position: 2)
-      self.pawns.create(x_position: column+1, y_position: 7)
+      self.pawns.create(x_position: column+1, y_position: 7, player_id: black_player_id)
```

Comment:
Currently this is putting the black player ID as "nil" since populate runs before the black player joins! We'll likely need to assign this later to avoid future color flipping!